mover-rclone/deployments/mysql/source/job.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: mysql-rclone
+spec:
+  template:
+    metadata:
+      name: mysql-rclone
+    spec:
+      containers:
+        - name: scribe-mover-rclone
+          image: quay.io/husky_parul/scribe-mover-rclone
+          command: ["sleep", "3600"]
+          env:
+            - name: RCLONE_CONFIG
+              value: /rclone-config/rclone.conf
+          volumeMounts:
+            # name must match the volume name below
+            - name: rclone-secret
+              mountPath: /rclone-config/
+              readOnly: true
+            - name: snapshot-data
+              mountPath: /var/lib/mysql/data
+      volumes:
+        - name: rclone-secret
+          secret:
+            secretName: rclone-secret
+        - name: snapshot-data
+          persistentVolumeClaim:
+          claimName: scribe-snap-pv-claim
```

Comment:
I believe the error is in regards to the indentation.
```

  volumes:
  - name: rclone-secret
    secret:
      defaultMode: 420
      secretName: rclone-secret
  - name: snapshot-data
    persistentVolumeClaim:
      claimName: scribe-snap-pv-claim
  - name: default-token-h57k8
    secret:
      defaultMode: 420
      secretName: default-token-h57k8
```

Without the spacing

```
  - emptyDir: {}
    name: snapshot-data
  - name: default-token-h57k8
```

Fix should be
```
        persistentVolumeClaim:
          claimName: snapshot-data
```

Reply:
Seems like a kube bug. The yaml does not complain about wrong indentation, instead mounts an EMPTY_DIR on the pod.